src/index.ts

```diff
@@ -0,0 +1,130 @@
+import * as path from "path"
+import * as fs from "@ts-common/fs"
```

Comment:
It looks like you had 8 character limit for the imports :D

Reply:
yeah, actually naming the imports a little more descriptively would help with readability